### ISSUE TITLE
Create generic fixtures for functional tests

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2756]
tl;dr: Create generic fixtures for functional tests to avoid confusion.

Details:
The functional tests right now sports specific number which relates (more or less) to real issues on the repo. This is historical, when our functional tests didn't have fixtures and were hitting directly the GitHub server for getting the JSON information. 

We should probably stop this. 

It creates confusion. @marimeireles for example wanted to exactly have the same rendering than what the original number seemed to be. What we want is to have a fixture which reflects what we need to test, not necessary what the actual server says for this number.

1. We could probably renamed the fixture, 1, 2, 3, 4, etc. That would help to identify more quickly things when it breaks. 
2. Make sure that the fixtures contains exactly what we need in terse and stable way. (be for failing or working).

**Comment by miketaylr** [comment]
It makes me think of #777. A small python script to scaffold out the bare minimum fixture file to be modified.
